components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java

```diff
@@ -246,6 +246,7 @@ public IdentityProviderResponse addIDP(IdentityProviderPOSTRequest identityProvi
 
         IdentityProvider identityProvider;
         try {
+            validateSystemReservedIDP(identityProviderPOSTRequest.getName());
```

Comment:
have to block this in idp name update as well

Reply:
Fixed by -https://github.com/wso2/identity-api-server/pull/616/commits/c92ae01802fec4c9966eec9e38aabb6399f293b8